docs/admin/docker.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 2
+id: docker
+title: Docker
+---
+
+# Docker
+**Docker**
+
+Spin up a temporary local instance of OpenCloud using **Docker**.
+
+
+**Prerequisites:**
+- **Linux**, **Mac** or **Windows** Subsystem for Linux [(WSL)](https://learn.microsoft.com/en-us/windows/wsl/install)
+- [**Docker**](https://docs.docker.com/compose/install/)
+
+
+---
+
+##  1. Create Required Directories for Bind Mounts
+
+```Shell
+mkdir -p $HOME/opencloud/opencloud-config
+mkdir -p $HOME/opencloud/opencloud-data
+```
+
+## 2. Pull OpenCloud Image
+
+```Shell
+docker pull opencloud-eu/opencloud:latest
+```
+
+## 3.  Initialize OpenCloud (First-time Setup)
+
+```Shell
+docker run --rm -it \
+    -v $HOME/opencloud/opencloud-config:/etc/opencloud \
+    -v $HOME/opencloud/opencloud-data:/var/lib/opencloud \
+    -e IDM_ADMIN_PASSWORD=admin \
+    opencloud-eu/opencloud:latest init
+```
+
+You can set your own password using `IDM_ADMIN_PASSWORD=your_password`. If not set, a password will be auto-generated
+
+<img src={require("./img/quick-guide/docker-opencloud-init.png").default} alt="Admin general" width="1920"/>
+
+
+## 4. Start OpenCloud
+
+```Shell
+docker run \
+    --name opencloud \
+    --rm \
+    -it \
+    -p 9200:9200 \
+    -v $HOME/opencloud/opencloud-config:/etc/opencloud \
+    -v $HOME/opencloud/opencloud-data:/var/lib/opencloud \
+    -e OC_INSECURE=true \
+    -e PROXY_HTTP_ADDR=0.0.0.0:9200 \
+    -e OC_URL=https://localhost:9200 \
+    opencloud-eu/opencloud:latest
+```
+
+## 4. Login
+
+Login with your browser:
+- [https://localhost:9200](https://localhost:9200)
+- user: **admin**
+- password: **admin**
+
+Congratulations! You’ve successfully set up and launched OpenCloud! Happy hacking!
```

Comment:
Should be rather at the end instead in the middle of the guide

Reply:
this is last step `4. Login` similar with quickstart guide

Reply:
<img width="857" alt="image" src="https://github.com/user-attachments/assets/30f05aa8-ad15-44fe-8781-ac97e14c4e8c" />

Still :/

Reply:
here is quickstart page. looks similar

![Screenshot 2025-02-18 at 16 22 17](https://github.com/user-attachments/assets/e5247300-5c26-45ec-b7d1-ed37cd8ba578)